apple/Sources/FerrostarMapLibreUI/Views/NavigationMapView.swift

```diff
@@ -22,17 +22,55 @@ public struct NavigationMapView: View {
     private var navigationState: NavigationState?
 
     @State private var locationManager = StaticLocationManager(initialLocation: CLLocation())
+
+    // MARK: Camera Settings
+
     @Binding private var camera: MapViewCamera
 
+    /// The snapped camera zoom. This is used to override the camera zoom whenever snapping is active.
+    @Binding private var snappedZoom: Double
+
+    /// Whether to snap the camera on the next navigation status update. When this is false,
+    /// the user can browse the map freely.
+    @Binding private var useSnappedCamera: Bool
+
+    /// The MapViewPort is used to construct the camera at the end of a drag gesture.
+    @State private var mapViewPort: MapViewPort?
+
+    /// The breakway velocity is used on the drag gesture to determine when allow a drag to
+    /// disable the snapped camera (assuming it's not constant(true).
+    ///
+    /// Tune this value to reduce the number of accidental drags that detach the camera
+    /// from the snapped user location.
+    private let breakwayVelocity: CGFloat
+
+    /// Initialize a map view tuned for turn by turn navigation.
+    ///
+    /// - Parameters:
+    ///   - styleURL: The style URL for the map. This can dynamically change between light and dark mode.
+    ///   - navigationState: The ferrostar navigations state. This is used primarily to drive user location on the map.
+    ///   - camera: The camera which is controlled by the navigation state, but may also be pushed to for other cases
+    /// (e.g. user pan).
+    ///   - snappedZoom: The zoom for the snapped camera. This can be fixed, customized or controlled by the camera.
+    ///   - useSnappedCamera: Whether to use the ferrostar snapped camera or the camer binding itself.
+    ///   - snappingBreakawayVelocity: The drag gesture velocity used to disable snapping. This can be tuned to prevent
+    /// accidental drags.
+    ///   - content: Any additional MapLibre symbols to show on the map.
     public init(
         styleURL: URL,
         navigationState: NavigationState?,
         camera: Binding<MapViewCamera>,
+        snappedZoom: Binding<Double> = .constant(18),
+        useSnappedCamera: Binding<Bool> = .constant(true),
+        snappingBreakawayVelocity: CGFloat = 25,
```

Comment:
This view is the most fundamental of 3 layers of hierarchy of nav views, but most users will interact directly with `DynamicallyOrientingNavigationView`.

1. We need to bubble up all of these new bindings and tunables :)
2. I'd suggest *not* having defaults on these more fundamental views. This does two things:
  a. Prevents us from making mistakes / forgetting to pass things down through the higher level layers.
  b. Makes it clear to end users that they are explicitly opting for the more fundamental views.

So these defaults should (IMO) move all the way up to `DynamicallyOrientingNavigationView`, and then the compiler will tell you what you missed in the plumbing if these don't have defaults.